process/economics/economicsData.go

```diff
@@ -215,6 +247,51 @@ func checkValues(economics *config.EconomicsConfig) error {
 		return process.ErrInvalidGasModifier
 	}
 
+	if len(economics.FeeSettings.GasLimitSettings) == 0 {
+		return process.ErrEmptyGasLimitSettings
+	}
+	conversionBase := 10
+	bitConversionSize := 64
+	for _, gasLimitSetting := range economics.FeeSettings.GasLimitSettings {
+		minGasLimit, err := strconv.ParseUint(gasLimitSetting.MinGasLimit, conversionBase, bitConversionSize)
```

Comment:
duplicated code on L256-L279 with the `setGasLimitSetting` function.

Reply:
It is different. This one is done in local variables just for checking the validity of the received parameters on New.. method, before the object is created. In setGasLimitSetting the convertion is done directly into members variables at epoch change. This conversion part could not be extracted into single method which satisfies both situations.

Reply:
Yes because the code is written in such way. Could have been refactored as to have only one parse function. I will make a PR that refactors this exact part and you will be one of the reviewers :D